Makefile

```diff
@@ -24,6 +24,9 @@ checkformat:
 format:
 	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) spotlessApply)
```

Comment:
We are adding pre-commit hook for spotless before every commit for other extensions.

https://github.com/adobe/aepsdk-edge-android/blob/dev-v3.0.0/Makefile#L19

https://github.com/adobe/aepsdk-edge-android/tree/dev-v3.0.0/.githooks
Can we add that for media.